pyproject.toml

```
@@ -0,0 +1,77 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "daedalus-deploy"
```

```suggestion
name = "daedalus_deploy"
```

needs to match the name of the modules you install I think - this is required for much of hatch to work for me